wsr-tic-tac-toe/src/components/square/index.tsx

```diff
@@ -23,6 +23,13 @@ const Square: React.FunctionComponent<SquareProps> = (props: SquareProps) => {
     }
     return squareState !== undefined ? (
         <div
+            aria-label={
+                squareState.selectedBy === null
```

Comment:
aria-label should be human readable and indicate what this element is.
In this case, it should be "square player 1", "square", "square player 2" and probably nothing extra for hovered

Reply:
Updated aria-label.